docs/en/reference/information_schema/collations.md

```diff
@@ -0,0 +1,18 @@
+---
+displayed_sidebar: "English"
+---
+
+# collations
+
+`collations` contains the available collations.
+
+The following fields are provided in `collations`:
+
+| **Field**          | **Description**                                              |
+| ------------------ | ------------------------------------------------------------ |
+| COLLATION_NAME     | The collation name.                                          |
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/3ee41377-9275-47d2-9e2a-bc7c0a36d861)
-> The name of the collation.